internal/clients/team2/social_capital.go

```diff
@@ -49,25 +56,41 @@ func (a *AgentTwo) updateTrustworthiness(agentID uuid.UUID, actualAction, expect
 
 }
 
-// func (a *AgentTwo) updateInstitution(agentID uuid.UUID) float64 {
+/// ///
+/// Networks
+/// ///
 
-// 	// return 0.5 // This is just a placeholder value
-// }
+func (a *AgentTwo) UpdateSocNetAgent(agentId uuid.UUID, amt float64, weight float64) {
```

Comment:
Don't think these two functions are being called anywhere right now so I couldn't interpret what 'amt' meant. Could you elaborate on what it is?

Reply:
@mc1017 Suggest I add it so we can update values when messages are received. `amt` is amount.